types/ext/settings.d.ts

```diff
@@ -119,6 +119,7 @@ export type GeneralOptions = {
     popupScaleRelativeToPageZoom: boolean;
     popupScaleRelativeToVisualViewport: boolean;
     showGuide: boolean;
+    contextMenu: boolean;
```

Comment:
I guess we might eventually do more stuff with the context menu, so could name this setting more descriptively, like `enableContextMenuScanSelected`.